aggregator/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.13
+version: 0.4.0
```

Comment:
@jay-bisonai Isn't there some better way so we don't need to change `version`, `appVersion`  and `tag`? Also, shouldn't all have the same value? What kind of impact it can have?

Reply:
![Screenshot 2023-11-14 at 9 59 29 AM](https://github.com/Bisonai/orakl-helm-charts/assets/137759552/9a1cb606-e2b5-4458-bd87-b0423fa1454b)
This value appear in user who using our helm chart.

The `version` is need to change when we changed structure of chart, or add some dependencies, or update in template directory or file.

The `appVersion` need to be same value for current version of app/service.

The `tag` is created image value in our image repository.

So, it doesn't matter if every value is different. I think only tag and appVersion values can be viewed as overlapping roles, but to solve this, I think we need to write code to generate a helm chart when there is an update, and it takes too much effort for the effect you can get.

Reply:
Thank you for explanation! I think we have to improve our code versioning process in Orakl Network 🙌